### ISSUE TITLE
Default units in driving region UK should be miles / yards rather than miles / meters

### ISSUE THREAD

**Issue by miklcct** [osmandapp/OsmAnd#11574]
### Description
The default units in the app in UK is miles / meters, but road signs in the UK are in miles / yards.

### How to reproduce?
Change the driving region to be the UK

### Your Environment
OsmAnd Version: 3.9.10
Android version: 11
Device model: Samsung Galaxy A42 5G

**Maps used (online or offline):**   
South West England

**Comment by Stephen-A-J** [comment]
Agree. Although trivial, I find this one of the most annoying features of Osmand. Why can we not either have the correct units for UK, or simply not have presets, but allow us to select Left/Right, Miles/km and Yds/metres/feet as three separate fields. Seems simpler than presets, and means that users are in control of what they really want.
